src/main/java/seedu/exercise/logic/parser/ExerciseBookParser.java

```diff
@@ -82,7 +82,7 @@ public Command parseCommand(String userInput) throws ParseException {
         case ScheduleCommand.COMMAND_WORD:
             return new ScheduleCommandParser().parse(arguments);
 
-        case CustomCommand.COMMAND_WORD:
+        case CustomAddCommand.COMMAND_WORD:
```

Comment:
The `COMMAND_WORD` should be in `CustomCommand`?

Reply:
Yes that was an oversight. Has made the changes.